packages/react/src/components/Pagination/Unstable_Pagination/Pagination.js

```diff
@@ -0,0 +1,277 @@
+/**
+ * Copyright IBM Corp. 2016, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import { settings } from 'carbon-components';
+import { CaretRight24, CaretLeft24 } from '@carbon/icons-react';
+import Button from '../../Button';
+import Select from '../../Select';
+import SelectItem from '../../SelectItem';
+
+const { prefix } = settings;
+
+function Unstable_Pagination({
```

Comment:
```suggestion
function unstable_Pagination({
```

Will try to make sure this is in our style guide soon! Totally get if the `U` was meant for usage in React components, I think we would just recommend renaming the import if that's the case:

```js
import { unstable_Pagination as Pagination } from '...';
```

Reply:
@joshblack I can update it to `unstable_` in all locations (including file names) if that's what you prefer?

Reply:
Yes please! The only reason we do this convention is to mirror FB's `unstable_` convention, e.g. `unstable_SuspenseList`. I'll make sure to add a reference to this in our style guide as well, sorry about the mixup

Reply:
Yeah no worries @joshblack I probably misremembered the capitalization when I wrote this

One thing I just noticed though, as I started to change to `unstable_` -- the linter is not pleased with me & flags hooks usage as a violation of `eslint(react-hooks/rules-of-hooks)`:

![Screen Shot 2020-06-01 at 10 13 35 AM](https://user-images.githubusercontent.com/9057921/83423441-9675c300-a3f0-11ea-88d0-6c8674430cdc.png)

> React Hook "useState" is called in function "unstable_Pagination" that is neither a React function component nor a custom React Hook function. eslint(react-hooks/rules-of-hooks)

How do you want to proceed here?